internal/flags.go

```diff
@@ -184,6 +184,11 @@ func NewApp() (app *cli.App) {
 				Usage: "Reduce S3 operation costs at the expense of some performance (default: off)",
 			},
 
+			cli.BoolFlag{
+				Name:  "no-dir-compat",
+				Usage: "Assume all directory objects are \"dir/\" (default: off)",
```

Comment:
I think this reads better:

Assume all directory objects ("dir/") exist (default: off)